vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,145 @@ sidebarDepth: 2
 
 ## Create the Project Structure
-First, create a parent project directory (e.g. `example-bundle`) along with a child bundle directory. In a project generated by the [Entando Component Generator](../../../docs/create/component-gen-overview.md), the microservice and micro frontend source files live under the parent directory.
 
-``` sh
-mkdir -p example-bundle/bundle; cd example-bundle/bundle
-```
+1. Create a parent project directory (e.g. `example-bundle`) and a child `bundle` directory:
+   ``` sh
+   mkdir -p example-bundle/bundle; cd example-bundle/bundle
+   ```
+
+   In a project generated by the [Entando Component Generator](../../../docs/create/component-gen-overview.md) (ECG), the microservice and micro frontend source files are stored in the parent directory.
 ## Add a Simple Widget
 
-Create a widget directory
-``` sh
-mkdir widgets
-```
-
-Create a widget descriptor file within that directory
-``` sh
-touch widgets/example-widget.yaml
-```
-
-Populate the widget descriptor file `example-widget.yaml` with a simple definition. Make sure to retain the correct YAML indentation of 2 or 4 spaces.
-``` yaml
-code: example-widget
-titles:
-   en: Example Widget
-   it: Widget d'esempio
-group: free
-customUi: <h2>Hi from Example Widget</h2>
-```
+1. Create a widget directory:
+   ``` sh
+   mkdir widgets
+   ```
+
+2. Create a widget descriptor file within that directory:
+   ``` sh
+   touch widgets/example-widget.yaml
+   ```
+
+3. Populate the widget descriptor file `example-widget.yaml` with a simple definition. Retain the correct YAML indentation of 2 or 4 spaces.
+   ``` yaml
+   code: example-widget
+   titles:
+      en: Example Widget
+      it: Widget d'esempio
+   group: free
+   customUi: <h2>Hi from Example Widget</h2>
+   ```
 
 ## Create the Bundle Descriptor
 
-The main file processed by the Entando Component Repository is `descriptor.yaml`, which describes all of the components within the bundle. The name of the bundle descriptor file must be `descriptor.yaml` and it must be stored in the child bundle directory (e.g. `example-bundle/bundle`).
-```sh
-touch descriptor.yaml
-```
-
-Populate the bundle descriptor file with the following YAML definition
-``` yaml
-code: example-bundle
-description: This is an example of an Entando bundle
-components:
-  widgets:
-    - widgets/example-widget.yaml
-```
-Component descriptor file names and locations (e.g. `widgets/example-widget.yaml`) are arbitrary since the bundle descriptor explicitly points to those files. Convention is to group components by type with all widgets in one directory, all page templates in another, etc.
+1. In the child bundle directory, create a bundle descriptor file named `descriptor.yaml`:
+   ```sh
+   touch descriptor.yaml
+   ```
+
+   All of a bundle's components are defined within `descriptor.yaml`, which is the main file processed by the ECR.
+
+2. Add the following definition to `descriptor.yaml`:
+   ``` yaml
+   code: example-bundle
```

Comment:
I think it had to be an underscore in the name in the final version but we haven't finished testing with a widget...